tower/CHANGELOG.md

```diff
@@ -7,16 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
-- **util**: Add `CloneBoxService` which is a `Clone + Send` boxed `Service`.
-- **util:** Add `ServiceExt::boxed` and `ServiceExt::clone_boxed` for applying the
-  `BoxService` and `CloneBoxService` middleware.
-- **builder:** Add `ServiceBuilder::boxed` and `ServiceBuilder::clone_boxed` for
-  applying `BoxService` and `CloneBoxService` layers.
-- **util**: Remove unnecessary `Debug` bounds from `impl Debug for BoxService`.
-- **util**: Remove unnecessary `Debug` bounds from `impl Debug for UnsyncBoxService`.
+- None.
+
+# 0.4.11 (November 18, 2021)
+
+### Added
+
+- **util**: Add `CloneBoxService` which is a `Clone + Send` boxed `Service` ([#615])
+- **util**: Add `ServiceExt::boxed` and `ServiceExt::clone_boxed` for applying the
+  `BoxService` and `CloneBoxService` middleware ([#616])
+- **builder**: Add `ServiceBuilder::boxed` and `ServiceBuilder::clone_boxed` for
+  applying `BoxService` and `CloneBoxService` layers ([#616])
```

Comment:
I know I should have spotted this in #616, but let me revive the naming question before a release goes out... I think think should be called `BoxCloneService` (and helper should be `boxed_clone`). My reasoning is that this makes the type/method names lexically grouped (i.e in documentation or tab completion). When searching I'll see all the `Box*Service`s together.

We can defer this to 0.5, though, I guess, since `CloneBoxService` is similar to `UnsyncBoxService`...

Reply:
@hawkw wondered about that too. I don't feel strongly about it, so think I'll just rename it in this PR.

Reply:
I think the change notes need to be updated as well, now...

Reply:
```suggestion
  applying `BoxService` and `BoxCloneService` layers ([#616])
```

Reply:
whoops! Was a bit too quick with the search and replace. Fixed it!